src/index.ts

```diff
@@ -2393,4 +2585,8 @@ export {
 	trim_decimal_overflow,
 	verifySignature,
 	resolveToENSName,
+	prepareGaslessDeposit,
+	pupulateGaslessDepositTx,
+	prepareGaslessReclaim,
+	populateGaslessReclaimTx,
```

Comment:
3 things that are different from the current codebase:

1. no createLink equivalent: wrapper function for createGaslessDeposit and reclaimLinkGasles that takes in an additional ssigner and then executes the function

Is it necessary? Not sure tbh. Can add to backlog. I think this is good as is and not sure there's a lot of need for it anyways. It's nice for devx, but I think this is good as is.

2. naming difference with prepareTxs. Here you call it populate (pupulate xd) GaslessDepositTx. We should standardize this and have the same naming convention across both. Populate seems like the better name choice, since we need a pre-stage before that and having preparePrepareTx is dumb. If we're going to use populate, then we should also rename prepareTxs to populateCreate

Reply:
Discussssssssssssśsšśsšśsšśsšśsšśsšśsšśsšśsšed in sync, renamed!